src/components/Welcome/index.jsx

```diff
@@ -34,8 +35,31 @@ export default class Welcome extends Component {
       return <div className="error">error here</div>;
     }
 
-    if (loading) {
-      return <div className="loading">loading .. </div>;
+    if (this.state.loading) {
```

Comment:
why you use this.state while you has destructor the loading in line 32

remove destructor or use it

Reply:
oops... I will do it